lib/linguist/languages.yml

```diff
@@ -1659,13 +1659,16 @@ HTML+Django:
   tm_scope: text.html.django
   group: HTML
   extensions:
-  - ".mustache"
   - ".jinja"
+  - ".mustache"
```

Comment:
Ah, please don't. We make an exception for the first extension in an extensions list, because that's what marks the "primary" (most common) extension. So what you've done here is make `.jinja` the primary extension of `HTML+Django`.